src/slick.grid.ts

```diff
@@ -4693,6 +4697,14 @@ export class SlickGrid<TData = any, C extends Column<TData> = Column<TData>, O e
       this.activeCellNode = this.getCellNode(this.activeRow, this.activeCell);
     }
   }
+  protected applyTopStyling(elements: NodeListOf<HTMLElement>) {
+    elements?.forEach((element: HTMLElement) => {
+      const top = element.dataset.top;
+      if (top) {
```

Comment:
yea it still gets set since the value is data-top value is set to 0px instead of 0, tho i agree it should be changed to !== undefined

Reply:
like mentioned in my other PR #894, after future code refactoring to remove dynamic html creation (with `x` and `xRight`), we could circle back to this and use `xRight.style.top` whenever we remove `innerHTML`, so I guess that would be ok for now since it should be temporary and get us closer to CSP compliance, however please read the other [post](https://github.com/6pac/SlickGrid/pull/908#discussion_r1385032302) I wrote up there